release-notes/5.0/5.0.0-preview1.md

```diff
@@ -0,0 +1,131 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 5.0.0-preview1.xxx released 9 March 2022
+
+This update brings the below changes over the previous release:
+
+### Added
+
+- Added SqlDataSourceEnumerator. [#1430](https://github.com/dotnet/SqlClient/pull/1430), [Read more](#sql-data-source-enumerator)
+- Added new attestation protocol `None` option to forgo enclave attestation when using VBS enclaves. [#1425](https://github.com/dotnet/SqlClient/pull/1425) and [#1419](https://github.com/dotnet/SqlClient/pull/1419), [Read more](#new-attestation-protocol-none)
+- Added a new AppContext switch to suppress insecure TLS warnings. [#1457](https://github.com/dotnet/SqlClient/pull/1457), [Read more](#suppress-insecure-tls-warning)
+
+### Fixed
+
+- Suppressed TLS security warning with `Encrypt = false` by new AppContext switch. [#1457](https://github.com/dotnet/SqlClient/pull/1457)
+- Fixed all documentation paths to Unix format path. [#1442](https://github.com/dotnet/SqlClient/pull/1442)
+- Fixed thread safety issue for `GetEnclaveProvider` by converting dictionary to concurrent dictionary. [#1451](https://github.com/dotnet/SqlClient/pull/1451)
+
+### Changed
+- Updated `Microsoft.Data.SqlClient.SNI` (.NET Framework dependency) and `Microsoft.Data.SqlClient.SNI.runtime` (.NET Core/Standard dependency) version to `v5.0.0-preview1.22062.1`. [#1537](https://github.com/dotnet/SqlClient/pull/1537)
+- modernized style in ValueUtilSmi. [#1351](https://github.com/dotnet/SqlClient/pull/1351)
+- Changed SQL server codenames to version names. [#1439](https://github.com/dotnet/SqlClient/pull/1439)
+- Prevented subtype generation in project files. [#1452](https://github.com/dotnet/SqlClient/pull/1452)
+- Changed `Array.Copy` to `Buffer.BlockCopy` for byte arrays. [#1366](https://github.com/dotnet/SqlClient/pull/1366)
+- Changed files in csproj to be alphabetically sorted in netfx and netcore. [#1364](https://github.com/dotnet/SqlClient/pull/1364)
+- TdsEnum, SqlInternalTransaction and MetaDataUtilsSmi are moved to shared folder. [#1369](https://github.com/dotnet/SqlClient/pull/1369), [#1346](https://github.com/dotnet/SqlClient/pull/1346) and [#1339](https://github.com/dotnet/SqlClient/pull/1339)
+
+### SQL Data Source Enumerator support
+Details on `Request`, `Response` and `timeouts` are available in [MC-SQLR](https://docs.microsoft.com/en-us/openspecs/windows_protocols/mc-sqlr/f9e7c304-ef7e-4d5b-9104-a3807c93436f)
```

Comment:
```suggestion
Provides a mechanism for enumerating all available instances of SQL Server within the local network.
```

Reply:
Thanks, it was pretty opaque what this is all about.